framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/scenario/builder/WorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder.java

```diff
@@ -38,15 +40,86 @@
 
     /**
      * Return setup builder with specified Smart router id.
+     *
      * @param smartRouterId Smart router id.
      * @return Builder with configured Smart router id.
      */
     WorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder withSmartRouterId(String smartRouterId);
 
     /**
-     * Return setup builder with specified timer service data store refresh interval.
-     * @param timerServiceDataStoreRefreshInterval timer service data store refresh interval.
-     * @return Builder with configured timer service data store refresh interval.
+     * Return setup builder with specified timer service data store refresh
+     * interval.
+     *
+     * @param timerServiceDataStoreRefreshInterval timer service data store
+     * refresh interval.
+     * @return Builder with configured timer service data store refresh
+     * interval.
      */
     WorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder withTimerServiceDataStoreRefreshInterval(Duration timerServiceDataStoreRefreshInterval);
+
+    /**
+     * Return setup builder with additional configuration for SSO deployment.
+     *
+     * @param deploySSO Default value is 'false'. Set to 'true' to deploy SSO to
+     * the scenario.
+     * @return Builder
+     */
+    WorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder deploySso(boolean deploySSO);
```

Comment:
Is that deploySSO parameter needed? Wouldn't it be sufficient to have just deploySso() ?